readme.md

```diff
@@ -66,6 +66,7 @@ It’s part of the [unified][] collective.
 * [eslint-plugin-markdown](https://github.com/eslint/eslint-plugin-markdown) - Lint JavaScript in markdown.
 * [documentation.js](https://github.com/documentationjs/documentation) - Documentation system for modern JavaScript.
 * [bytemd](https://github.com/bytedance/bytemd) - Hackable markdown editor component.
+* [Astro](https://astro.build) - Build faster websites with Astro's next-gen island architecture 🏝✨
```

Comment:
```suggestion
* [Astro](https://astro.build) - Build faster websites with Astro’s next-gen island architecture.
```

- Straight apostrophe to smart apostrophe
- I think the emoji art is a bit much, it’s fun, but that’s not really what this page is for